activities/TankOp.activity/js/activity.js

```diff
@@ -38,30 +38,19 @@ define(["sugar-web/activity/activity"], function (activity) {
 					app.load();
 					app.renderInto(document.getElementById("board"));
 
-					function zoomin(){
-						var x=document.getElementsByClassName("body");
-						for (var i = 0; i < x.length; i++) {
-							x[i].style.bottom="40px";
-						} 
-					}
-
-					function zoomout(){
-						var x=document.getElementsByClassName("body");
-						for (var i = 0; i < x.length; i++) {
-							x[i].style.bottom="0px";
-						} 
-					}
+					play= new TankOp.Play();
```

Comment:
Not a good idea to create a new Play object. Play object is the play screen so it should be created only when the play screen is displayed.